themes/default/theme/src/scss/_search.scss

```diff
@@ -1,184 +1,55 @@
 .docs-type-nav-search {
-    display: flex;
-    flex-direction: column-reverse;
-    justify-content: space-between;
-    background-color: $white;
+    @apply flex flex-col-reverse justify-between bg-white;
```

Comment:
can you leave the css as it was instead of changing to tailwind unless we have to?

Reply:
I'd prefer we kept to the standard Tailwind here, as it eliminates a bunch of code that Tailwind covers -- but I don't at all mind you writing CSS if that's what makes you happy!

Reply:
@cnunciato we talked in the last frontend working group that docs is moving away from tailwind, its all mostly been removed. id prefer if you didn't slowly add it back.